python/pyarrow/tests/test_parquet.py

```diff
@@ -2170,7 +2170,10 @@ def s3_example_s3fs(s3_connection, s3_server, s3_bucket):
 
     fs.mkdir(bucket_uri)
     yield fs, bucket_uri
-    fs.rm(bucket_uri, recursive=True)
+    try:
+        fs.rm(bucket_uri, recursive=True)
+    except FileNotFoundError:
+        pass
```

Comment:
For some reason the teardown gave errors for the added test where nothing got written (because we already errored when checking the file path). Not sure why though.

It gave `FileNotFoundError: s3://test-s3fs/07f5f3460595484aae4482f19168c3cc`

Reply:
This is using `s3fs`. I don't think we care about such errors.